SciGraph-core/src/main/java/io/scigraph/internal/TinkerGraphUtil.java

```diff
@@ -157,32 +189,32 @@ static Edge addEdge(Graph graph, Edge edge) {
   }
 
 
-  public static Element addElement(Graph graph, Element element) {
+  public Element addElement(Element element) {
     if (element instanceof Vertex) {
-      return addNode(graph, (Vertex) element);
+      return addNode((Vertex) element);
     } else {
-      return addEdge(graph, (Edge) element);
+      return addEdge((Edge) element);
     }
   }
 
-  public static void addGraph(Graph graph, Graph addition) {
+  public void addGraph(Graph addition) {
     for (Vertex vertex : addition.getVertices()) {
-      addElement(graph, vertex);
+      addElement(vertex);
     }
     for (Edge edge : addition.getEdges()) {
-      addElement(graph, edge);
+      addElement(edge);
     }
   }
 
-  public static Graph combineGraphs(Graph graph1, Graph graph2) {
-    Graph graph = new TinkerGraph();
-    addGraph(graph, graph1);
-    addGraph(graph, graph2);
-    return graph;
+  public Graph combineGraphs(Graph graph2) {
+    TinkerGraphUtil tgu = new TinkerGraphUtil(curieUtil);
+    tgu.addGraph(graph);
+    tgu.addGraph(graph2);
+    return tgu.getGraph();
   }
 
-  public static TinkerGraph resultToGraph(Result result) {
-    TinkerGraph graph = new TinkerGraph();
+  public Graph resultToGraph(Result result) {
```

Comment:
Making this static is tricky because it's making several non-static calls to addElement which in turn make non-static calls to addNode and addEdge. I would have to make static versions of those functions. It's simpler to keep resultToGraph non-static. Let me know if you prefer it static and I will rewrite it that way.